packages/terra-image/src/Image.module.scss

```diff
@@ -36,4 +36,29 @@
   .hidden {
     display: none;
   }
+
```

Comment:
I think it's worth adding inline documentation here explaining why `font-family` is introduced and why it has 'object-fit'.

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2494/commits/0127dec949032aa00c86bb77ebdeb92d3d695832.